packages/at_secondary_server/lib/src/verb/handler/notify_verb_handler.dart

```diff
@@ -71,7 +71,8 @@ class NotifyVerbHandler extends AbstractVerbHandler {
       var currentAtSign = AtSecondaryServerImpl.getInstance().currentAtSign;
       // If '@' is missing before an atSign, the formatAtSign method prefixes '@' before atSign.
       verbParams[FOR_AT_SIGN] = AtUtils.formatAtSign(verbParams[FOR_AT_SIGN]);
-      verbParams[AT_SIGN] = AtUtils.formatAtSign(verbParams[AT_SIGN]);
+      verbParams[AT_SIGN] =
+          AtUtils.formatAtSign(verbParams[AT_SIGN]) ?? currentAtSign;
```

Comment:
Srieteja recently merged a PR. since there were two methods formatAtSign and fixAtSign, I suggested to deprecated formatAtSign and use fixAtSign. Please uptake latest version of at_utils and change this method to fixAtSign
https://github.com/atsign-foundation/at_tools/pull/300

Reply:
Sure Murali.

Reply:
@murali-shris: I did look for the usages and there are around 20 usages in at_secondary_server. Is it fine to do these changes in a separate PR?